src/actions/AuthActions.js

```diff
@@ -94,23 +94,23 @@ export const authenticate = () => async dispatch => {
   }
 
   /*
-    Finally, we make sure that we are able to connect to the Linnia Smart contracts. We use 
+    Finally, we make sure that we are able to connect to the Stow Smart contracts. We use 
     Web3 to ping the contract address. If we receive and empty hex, we have provided the wrong
     hub address. This is another developer check.
   */
 
   const code = await web3.eth.getCode(hubAddress);
   if (!code || code === '0x0' || code === '0x') {
-    console.error('Linnia is not configured correctly!');
-    return dispatch(authFailure(LINNIA_MISCONFIGURED));
+    console.error('Stow is not configured correctly!');
+    return dispatch(authFailure(STOW_MISCONFIGURED));
   }
 
-  const linnia = new Linnia(web3, { linniaContractUpgradeHubAddress: hubAddress });
+  const stow = new Stow(web3);
```

Comment:
Should still be something like 

`const stow = new Stow(web3, { hubAddress })` so user can optionally pass in hubAddress when developing locally